app/service/file_svc.py

```diff
@@ -78,15 +89,32 @@ def __init__(self, plugins, exfil_dir):
         :param name:
         :return: a tuple (file_path, contents)
         """
-        _, file_name = await self.find_file_path(name, location='payloads')
-        if file_name:
-            with open(file_name, 'rb') as file_stream:
-                return name, file_stream.read()
-        _, file_name = await self.find_file_path('%s.xored' % (name,), location='payloads')
-        if file_name:
-            return name, xor_file(file_name)
+        for loc in ['payloads', 'data']:
```

Comment:
instead of looping through a list of possible locations, we should determine a required location to put payloads. either data/payloads or payloads/ . Offhand it seems like the later is more suitable.. thoughts?

Reply:
I'd prefer data because it falls inline with all the other stuff